src/main/java/pl/widulinski/webcrawlertool/searchData/DataToSearch.java

```diff
@@ -0,0 +1,25 @@
+package pl.widulinski.webcrawlertool.searchData;
+
+
+import lombok.Data;
+import org.springframework.stereotype.Component;
+import pl.widulinski.webcrawlertool.enums.Categories;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import java.util.Set;
+
+
+
+@Component
+@Data
+public class DataToSearch {
+
+    private String shop;
+
+    @Enumerated(EnumType.STRING)
```

Comment:
o ile tej klasy nie zapisujesz w bazie to tutaj tez annotacja z pakietu `persistance` niewiele daje